src/responsibleai/rai_analyse/rai_component_utilities.py

```diff
@@ -77,16 +77,42 @@ def load_mlflow_model(
     mlflow.set_tracking_uri(workspace.get_mlflow_tracking_uri())
 
     if model_id:
-        model = Model._get(workspace, id=model_id)
+        try:
+            model = Model._get(workspace, id=model_id)
+        except Exception as e:
+            raise UserConfigError("Unable to retrieve model by model id {} in workspace {}, error:\n{}".format(model_id, workspace.name, e))
         model_uri = "models:/{}/{}".format(model.name, model.version)
 
     if use_model_dependency:
-        pip_file = mlflow.pyfunc.get_model_dependencies(model_uri)
-        subprocess.check_call([sys.executable, "-m", "pip",
-                                "install", "-r", pip_file])
+        try:
+            pip_file = mlflow.pyfunc.get_model_dependencies(model_uri)
+        except Exception as e:
+            raise ValueError("Failed to get model dependency from given model {}, error:\n{}".format(model_uri, e))
+        try:
+            subprocess.check_output([sys.executable, "-m", "pip",
+                                    "install", "-r", pip_file])
+        except subprocess.CalledProcessError as e:
+            _logger.error("Installing dependency using requriments.txt from mlflow model failed: {}".format(e.output))
```

Comment:
will we get info on what installation specifically failed and why?

Reply:
e.output is the pip install log error and it is shown to user. Theres a separate classify and log method called after this